cmd/shortener/example_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"net/http/httptest"
+
+	"github.com/PerfectStepCoder/shorturl/internal/handlers"
+	"github.com/PerfectStepCoder/shorturl/internal/storage"
+	"github.com/go-resty/resty/v2"
+)
+
+const (
+	exampleLengthShortURL = 10
+	exampleBaseURL        = "http://localhost:8080"
+)
+
+func ExampleTestShorterURL() {
```

Comment:
В примерах строка `Test` меняется на `Example`, иначе пример не будет отображаться в документации.
```suggestion
func ExampleShorterURL() {
```